tests/resources/metadata/sample_metadata1.json

```diff
@@ -36,14 +31,23 @@
         ],
         "links": [
             {
-                "sample1": "http://data.ceda.ac.uk/badc/ukcp09/data/gridded-land-obs/gridded-land-obs-daily/"
+              "name": "example model",
+              "url": "https://drive.google.com/open?id=1uuz50RGiAW8YxRcWeQVgQglZpyAebgSM"
             },
-
             {
-                "sample2": "http://data.ceda.ac.uk/badc/ukcp09/data/gridded-land-obs/gridded-land-obs-averages-25km/"
+              "name": "example code",
+              "type": "example code",
+              "url": "https://github.com/slothkong/CNN_classification_10_monkey_species"
             },
             {
-                "fieldsDescription": "http://data.ceda.ac.uk/badc/ukcp09/"
+              "url": "https://s3.amazonaws.com/datacommons/links/discovery/n5151.jpg",
+              "name": "n5151.jpg",
+              "type": "discovery"
+            },
+            {
+              "url": "https://s3.amazonaws.com/datacommons/links/sample/sample.zip",
+              "name": "sample.zip",
+              "type": "sample"
```

Comment:
Regarding all the deltas to this file... do they also want to be applied to the file under examples...?
`$ diff -u tests/resources/metadata/sample_metadata1.json examples/sample_data/sample_metadata.json `

NOTE: the program `./examples/register_asset.py` seems to work (without showing any errors).

Reply:
I noticed at the moment squid-py does not seem to enforce a strict version of the meta data, but `plecos` does with `validate`. so yes I should change this as well